MarkupHandler.inc.php

```diff
@@ -324,11 +324,17 @@ protected function _buildMediaInfo($request, $assets)
 		$infos = array();
 		$mediaDir = 'markup/media';		# TODO Where to fetch media images from in OJS?
 		$context = $request->getContext();
+		$router = $request->getRouter();
+		$dispatcher = $router->getDispatcher();
 		foreach ($assets as $asset) {
-			$path = $asset['path'];
+			$path = str_replace('media/', '', $asset['path']);
 			$filePath = "{$mediaDir}/{$path}";

```

Comment:
Also dead code?

Reply:
Woops, there was one more thing to do: Display file metadata (see https://github.com/kaschioudi/ojs3-markup/pull/58/commits/6126ad10d14b1c61f74d919804665befacb77ed7)